.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
```

Comment:
Could you add a comment on the top describing the use case of the file?

Reply:
Done.